README.md

```diff
@@ -136,6 +121,85 @@ see specs in `spec/` folder:
     expect(mail_stub.with_args('-s', 'hello').stdin).to eql 'world'
   end
 ```
+### Test entire script
+
+```ruby
+let(:stubbed_env) { Rspec::Shell::Expectations::StubbedEnv.new }
+stubbed_env.execute('./spec/scripts/function_library.sh')
```

Comment:
This might not be the best example, as that script doesn't actually execute anything :)